.circleci/config.yml

```diff
@@ -28,7 +28,7 @@ commands:
       - run:
           name: Update globally available npm to the latest version
           # Note: We need npm >= 8.3 which supports 'overrides' in package.json
-          command: sudo npm install npm --global
+          command: npm install npm --global
```

Comment:
In the new `cimg/` images this not only does not require `sudo` but actually does not work at all with it:
```
sudo: npm: command not found
```